### ISSUE TITLE
Integrated terminal - links are not highlighted/clickable

### ISSUE THREAD

**Issue by mkuratczyk** [Microsoft/vscode#35552]
- VSCode Version: 1.17.0-insider
- OS Version: macOS 10.12.6

Steps to Reproduce:

1. display some links in the integrated terminal
2. hover over with CMD pressed
3. expected - links to be underlined and clickable but they are not

In the latest Insider build no links are clickable for me - either path nor http links. I checked 1.16.1 on the same system and at least http links are clickable but paths are not either.

Reproduces without extensions: Yes

**Comment by Tyriar** [comment]
@mkuratczyk can you provide a screenshot of the link(s) that are not working?

Also the commit hash of the VS Code version you're on would be useful, you can see this in the about dialog, eg. 5316a2d

**Comment by mkuratczyk** [comment]
Hey. I just updated to the latest Insider build. I don't see the commit hash there but it says:
Version 1.17.0-insider (1.17.0-insider)
81fc22ec957de26d2bf6aada3f16f04e0164741c
2017-10-03T22:43:34.772Z

I recorded a short video to show the problem: https://youtu.be/6W5ZWXwV2nQ
I don't see any determinism here. One path actually gets highlighted and clicking it takes me to the file as expected (so do the http link that get highlighted) but still some links/path do not get highlighted.
